src/main/java/org/hyperledger/fabric/sdk/ChaincodeEndorsementPolicy.java

```diff
@@ -239,7 +239,9 @@ public void fromFile(File policyFile) throws IOException {
     @Deprecated
     public void fromYamlFile(File yamlPolicyFile) throws IOException, ChaincodeEndorsementPolicyParseException {
         final Yaml yaml = new Yaml(new SafeConstructor());
-        final Map<?, ?> load = (Map<?, ?>) yaml.load(new FileInputStream(yamlPolicyFile));
+        FileInputStream policyFileStream = new FileInputStream(yamlPolicyFile);
```

Comment:
Could you use try-with-resources to ensure the close is called?
https://docs.oracle.com/javase/tutorial/essential/exceptions/tryResourceClose.html